Makefile

```diff
@@ -41,12 +31,28 @@ fmt:  ## Run autoformatting and linting
 .PHONY: pre-commit
 pre-commit: fmt  ## Run all code quality checks
 
+ifndef SCALE_FACTOR
+
+data/tables/.generated:
+	@echo "SCALE_FACTOR not set, skipping data table generation"
+	@touch $@
+
+data/tables/:
```

Comment:
When I call `SCALE_FACTOR=1 make data/tables/`, I get tables in `data/tables/scale-1/`. But when running the benchmarks, the tables are expected to be in `data/tables/scale-1.0/`.

Can handle the formatting for either way works?